go-controller/pkg/ovn/pods.go

```diff
@@ -109,21 +109,22 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod) {
 	portInfo, err := oc.logicalPortCache.get(logicalPort)
 	if err != nil {
 		klog.Errorf(err.Error())
-		// If ovnkube-master restarts, it is also possible the Pod's logical switch port
-		// is not readded into the cache. Delete logical switch port anyway.
-		err = ovnNBLSPDel(oc.nbClient, logicalPort, pod.Spec.NodeName)
-		if err != nil {
-			klog.Errorf(err.Error())
-		}
 
-		// Even if the port is not in the cache, IPs annotated in the Pod annotation may already be allocated,
-		// need to release them to avoid leakage.
 		logicalSwitch := pod.Spec.NodeName
```

Comment:
lets just get rid of this logicalSwitch variable. I don't think it adds anything useful.